### ISSUE TITLE
Sustainable Travel Support for WCEU24

### ISSUE THREAD

**Issue by gusaus** [WordPress/sustainability#11]
This item has been created to track the development of support information to encourage and enable visitors to WCEU24 in Turin to use the most sustainable travel options appropriate. 
We want to share information in a clear, accessible, and inclusive way to enable attendees to make the best choices they can about travelling to the city and the venue.

Draft Usecase:
As a WCEU24 attendee I want to easily understand what my options are for travel to the event, and compare them in terms of travel time, accessibility needs, cost, and environmental impact.

See [Google doc](https://docs.google.com/document/d/1-BJSJ2KYTyTVZnfQe1SmU1TgGGlkMPjDya8XzCoa-G8/edit#heading=h.j301qextlb2b).

**Comment by YellowlimeNL** [comment]
Who's willing to help with a graphic design for the WCEU Torino website?

**Comment by YellowlimeNL** [comment]
[The Google doc about travelling to WCEU in a sustainable way](https://docs.google.com/document/d/1-BJSJ2KYTyTVZnfQe1SmU1TgGGlkMPjDya8XzCoa-G8/edit#heading=h.j301qextlb2b) contains already relevant info. Possibly too much information to put at https://europe.wordcamp.org/2024/travelling-to-torino/.

My suggestions would be:

1. [WCEU site](https://europe.wordcamp.org/2024/travelling-to-torino/) shows brief info + infographic.

2. We publish a post at https://make.wordpress.org/sustainability/ (maybe later a handbook item) with more background info about sustainable travel in general and to Torino in particular.

What do you think?

**Comment by ramonfincken** [comment]
ad 1:

Yes we need to convert the doc to a neat summary with urge to avoid (where possible) planes and non-carshare.
If none is available I am willing to ask (and pay) my network for a graphic.

**Comment by YellowlimeNL** [comment]
Two contributors from #design team have offered their help with the graphic.

Update: I was advised to contact WCEU's Design team first. Will do.

**Comment by meeware** [comment]
ChooCHoo page is now here:
https://choo-choo.eu/wceu24/

This page includes the following:

- Intro text describing what the page includes in terms of information, and interactivity.
- A list of routes, including timings and costs, to the event based on published train timetables from 14 cities across europe. 
- a form at the bottom labeled 'Your Congress Journey' with dropdowns for journey type, departure stations, numnber of travelers, departure and return dates, plus a free text field for billing information. 

![screencapture-choo-choo-eu-wceu24-2023-11-24-12_44_47](https://github.com/WordPress/sustainability/assets/1597989/5f4d8fd7-c5ad-402a-9e6f-c08d524c4e70)

**Comment by meeware** [comment]
Issues with this page:

It is not clear what the function of this page is, nor how a user is to interact with it. I really don't understand what the 'user stories' might be, so it's very hard to test against those. 

If the user story is 'as an attendee to WCEU I can see what my options are for travel to the event by train' then there are issues as follows:

The list doesn't include the 14 largest cities in europe, it includes... 14 big cities. Excluding Moscow might be understandable, but Glasgow? What's the logic here? Why only 14, and why these 14?

The decision to simply list these as a flat HTML page places a limit on the navigability and findability of the data. IF we want to add more data, how can we?

The choice to only display the weekday train times is weird - WCEU finishes on a Saturday. It's highly likely that a large number of attendees will wish to travel home on a sunday - that information is likely important to atendees.

What booking services are available? it's not clear what booking services can be provided, in what timescale or what the steps involved are.

The design of the booking interface has profound issues: What billing information is required? Why? How is it validated? where is it stored? What gdpr measures are in place? How will it react if users enter credit cards? This is a massive MASSIVE red flag from a service design and security POV. 

We should do this afresh, define user journeys, and get something built that is clear in the functionality it delivers, navigable, includes data that is relevant to WCEU attendees, and which links to a properly designed and built booking system with proper support information available in (at least) EFIGS. 

EDIT - Additional problem. Many of the routes proposed assume rail travel between Paris Gare de Lyon and Turin. This route is closed owing to a landslide in September 2023 and the route is not expected to reopen until the end of 2024. Routes for Amsterdam, Bordeaux, Brussels, London, Lyon, and Paris are not valid.

**Comment by meeware** [comment]
> [The Google doc about travelling to WCEU in a sustainable way](https://docs.google.com/document/d/1-BJSJ2KYTyTVZnfQe1SmU1TgGGlkMPjDya8XzCoa-G8/edit#heading=h.j301qextlb2b) contains already relevant info. Possibly too much information to put at https://europe.wordcamp.org/2024/travelling-to-torino/.
> 
> My suggestions would be:
> 
> 1. [WCEU site](https://europe.wordcamp.org/2024/travelling-to-torino/) shows brief info + infographic.
> 2. We publish a post at https://make.wordpress.org/sustainability/ (maybe later a handbook item) with more background info about sustainable travel in general and to Torino in particular.
> 
> What do you think?

I think we should take the existing WCEU page, and propose an edit to it. 
We can have links to resources (euro rail and trainline and maybe DB, _possibly_ the ChooChoo page, though I have big issues with that) and enhanced information about how to choose a sustainable travel option, and tips for maximising the sustainability of all options (car share, keep up tyre pressures, train not taxi to the airport etc) We should put a sef imposed limit on the wordcount for the text we propose to add to that page.

**Comment by ohiawins** [comment]
The efforts being put into responsible travel by the SUS team here are brilliant. THANK YOU SO MUCH! 

I took a peak at some of these links. ChooChoo does bring up some concerns in the billing field and I tried to dig into it further but the English link in the menu doesn't seem to be working for me. The man in seat 61 is JAM-PACKED with info and a little overwhelming to sift through (I would love to get my hands on it for a redesign). Have we thought about links to https://www.rome2rio.com/ or https://www.raileurope.com? Again thank you all for infusing this into WCEU. So super important for us to strive to make as little environmental impact as possible. 

What about encouraging chartering busses? Like a bus full of WordCampers from the major cities to Turin? And asking people who road-tripped last year to share their experiences to encourage cars and not plane. I know Remkus drove with his son to WCEU and seemed to have a good time. 

Question: Does WCEU have a telegram channel for something like ride-shares? In our initial town-hall meeting I suggested something like this in the P2 brainstorming document. Does anyone here have access to that document to see what WCEU organizers were dreaming up back in October?

**Comment by ramonfincken** [comment]
WordCamp germany (this year) had a special page with comments where you could enter your contact info for care shares.

**Comment by meeware** [comment]
2023-11-28 Rail travel to Turin Update

TL:DR - both the western rail route to France and the northern rail route to Germany will be closed in June 2024 resulting in potentially longer, more complex and in many cases more expensive rail journeys for travellers to WCEU from North and Northwestern Europe. I'll be looking at the detailed impact of these changes as the timetables are announced in December, and also working out good alternative options for attendees.


![Alp train connections ](https://github.com/WordPress/sustainability/assets/1597989/b56613ef-1ce9-40ef-9832-905628d6d771)



Detail:
[The Fréjus Tunnel](https://en.wikipedia.org/wiki/Fr%C3%A9jus_Rail_Tunnel): The main rail line from Paris Gare de Nord to Turin and onward to Italy runs through this tunnel. On the ground it runs from Modane in France to Bardonecchia in Italy under Mont Cenis. In August of 2023  massive landslide on the french side, initially sending thousands of tinnes of rock onto the tracks. Subsequent investigations have revealed massive instability above the rails, and the current estimate is that the line won't reopen until late 2024 at the earliest. 
This line closure affects all rail travel from points west of Cologne and north of Bordeaux into Turin. Tickets along this route now change to a coach over the mountain pass, and involve an additional 1 to 4 hours travel, additional changes (with associated risk of missed connections) and low cost tickets are no longer available.

[The Gothard Tunnel](https://en.wikipedia.org/wiki/Gotthard_Base_Tunnel): The main rail line from Zurich to Milan runs through a tunnel complex that runs from Erstfeld in Switzerland to Biasca, also in Switzerland. It's the worlds longest rail tunnel. In August 2023 there was a freight train derailment that caused extensive damage inside the tunnel and services are running at a reduced level currently. In June 2024 the tunnel will close for repairs, currently scheduled to complete in September 2024. Timetable changes have not yet been published, but we anticipate very significant disruption to rail services to points as far east as Stuttgart. 

There will be trains along the coastal route (crossing at Ventimiglia) but this is not a high speed line. It's achingly beautiful but not fast at all. The Sempione crossing  (from Bern to Milan)is also expected to be open; again beautiful, again slow. And the Brennero route is expected to be open, a high speed line running basically from Munich down to Verona. 

Given the multiple closed routes I recommend the following:

- We make clear that rail travel to the event is possible, but a bit more of an adventure than usual for travellers from NW europe, and they should expect to have to plan carefully - we should point them at high quality information sites like [The Man in Seat 61](https://www.seat61.com/Italy.htm#london-to-italy-by-eurostar-and-tgv) - and the Choochoo page as proposed is not appropriate.
- We let the WCEU organisers know about these issues now. This demonstrates we are on top of good transport information, and sets them up to get information from us.
- We explore coach travel options for NW europe locations - country by country (UK, France, Ireland, Benelux, Western areas of Germany, Denmark) and try and find local reps in each to liaise with.

**Comment by meeware** [comment]
Current WCEU travel Page: https://europe.wordcamp.org/2024/travelling-to-torino/

Recommended changes (feedback from the team please):
Rail
Torino is very well linked to the Italian railway system, and usually to the rest of the european high speed network. Usually.

High-speed Frecciarossa trains are used nationally and connect Torino with Milan in 44 minutes and Rome in just over 4 hours.

Rail travel to Northern and Western Europe will be significantly disrupted during the period of WCEU owing to closure of the main tunnels under the Alps to France and Germany (Via Switzerland). Timetable changes have not yet been published, but we anticipate higher costs, longer journeys, and more changes. The routes that remain open are stunningly beautiful along the coast, and through high mountain passes, and we still encourage attendees to consider these options if traveling from western and Northern europe. 

The following sites will be great resources for anyone wanting to book tickets. Tickets are generally on sale 60 to 30 days before travel:
[The Trainline](https://www.thetrainline.com/)
[Rail Europe](https://www.raileurope.com/
)
[Interrail-eurial](https://www.interrail.eu/en)
 
For general rail travel information these three sites offer great advice:
[The Man In Seat 61](https://www.seat61.com/index.html)
[Rome 2 Rio](https://www.rome2rio.com/) (Also gives flight and coach options)[
Show Me The Journey](https://showmethejourney.com/)

For people coming from points east of Munich, rail travel offers good high speed services and is expected to be operating well.

**Comment by YellowlimeNL** [comment]
Tx for the valuable research and feedback @meeware! Based on your input, I've given a start to [reformulate the text proposal for the WCEU travel page](https://docs.google.com/document/d/1-BJSJ2KYTyTVZnfQe1SmU1TgGGlkMPjDya8XzCoa-G8/edit?disco=AAABBQiKmkI). 

Everyone's feedback and input is welcome.

**Comment by YellowlimeNL** [comment]
I made some changes. Is [text proposal](https://docs.google.com/document/d/1-BJSJ2KYTyTVZnfQe1SmU1TgGGlkMPjDya8XzCoa-G8/edit?disco=AAABCEp6Q9A) ready to be shared with the WCEU content team?

With regards to the (info)graphic. Uncertainties about routes and knowing that time schedules have not been published yet. I suggest to postpone the creation of the graphic to a later date.

**Comment by ramonfincken** [comment]
I agree on the postpone.

Made some additions to the google doc

**Comment by YellowlimeNL** [comment]
[New Gdoc ](https://docs.google.com/document/d/1ihntEsCTcWDTa3DaGhIpDkZkdTUR5tazt8-EAn23WgA/edit?usp=sharing)has been created with text proposal for better overview.

Thank you in advance for having a look at the text prior to sharing them with the Communication Team

**Comment by YellowlimeNL** [comment]
The Travelling to [Torino page at the WCEU 2024 website](https://europe.wordcamp.org/2024/travelling-to-torino/) has been updated. Thanks for the input by so many.

If you feel the page needs more info to make it more up to date or complete. Please mention it in the [Gdoc](https://docs.google.com/document/d/1ihntEsCTcWDTa3DaGhIpDkZkdTUR5tazt8-EAn23WgA/edit?usp=sharing). That doc is monitored and input added will be shared with the WCEU local team.
